### ISSUE TITLE
A question about the accuracy of your monodepthv2 reimplementation

### ISSUE THREAD

**Issue by LiuJF1226** [ZM-Zhou/SMDE-Pytorch#3]
Hi !  Thank you for your solid work. I have noted that the performance of your monodepthv2 reimplementation listed in your table is much better than that listed in the monodepthv2 paper (Digging Into Self-Supervised Monocular Depth Estimation),  especially for the R18+320*1024+mono configuration. Did you conduct something different, like the hyper-parameters or others ?  Hope to your reply.
![image](https://user-images.githubusercontent.com/34789037/226090372-7d287836-63c3-4b57-b558-b989b9704636.png)
![image](https://user-images.githubusercontent.com/34789037/226090390-ddc7bdad-133e-4784-951a-05340f834cf3.png)

**Comment by ZM-Zhou** [comment]
Hi Sauf4896,
In my opinion, I didn't intentionally change any settings. One difference is that the results I reported are calculated with the model with the best performance in all epochs, while those in the official repo are calculated with the last model after training. However, I think the more important reason may be the randomness in training. You could repeat the experiment several times in this repo and the official one and should get more consistent results.

**Comment by LiuJF1226** [comment]
Thanks to your reply! Recently I haved noted that a difference setting is that in your training script, you train the 320*1024 model for 5 epochs from the 192*640 10th epoch model. The official repo does train it from scratch for 20 epochs. Does this matter?

**Comment by ZM-Zhou** [comment]
I think it could be one of the main reasons.  My training setting is cited from their Supplementary Material: 

> "Our high resolution models were initialized using the weights from our standard resolution (640 x 192) model after 10 epochs of training. We then trained our high resolution models for 5 epochs with a learning rate of 10^-5. We used a batch size of 4 to enable this higher resolution model to fit on a single 12GB Titan X GPU."

I didn't notice that the setting in the official repo was different from it.

**Comment by LiuJF1226** [comment]
I got it. Thank you very much.
